cms/envs/common.py

```diff
@@ -1490,6 +1490,7 @@
     'experiments',
 
     'openedx.core.djangoapps.external_user_ids',
+    'openedx.core.djangoapps.demographics',
```

Comment:
Do we need this installed in studio? I think it's only being called from LMS and it's not being imported by other apps that are in studio.

Reply:
Remove it from the CMS.